core/src/test/java/org/elasticsearch/index/query/BaseQueryTestCase.java

```diff
@@ -283,7 +308,9 @@ public void testSerialization() throws IOException {
      * @return a new {@link QueryParseContext} based on the base test index and queryParserService
      */
     protected static QueryParseContext createContext() {
-        return new QueryParseContext(index, queryParserService);
+        QueryParseContext queryParseContext = new QueryParseContext(index, queryParserService);
+        queryParseContext.parseFieldMatcher(ParseFieldMatcher.EMPTY);
```

Comment:
it prevents an NPE because we don't set the ParseFieldMatcher in our test context otherwise.